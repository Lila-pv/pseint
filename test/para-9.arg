--protect_for_counter=1
