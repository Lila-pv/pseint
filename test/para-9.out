1
=== Line 4: ExeError 322
